composer.json

```diff
@@ -22,7 +22,7 @@
         "symfony/property-access": "~2.3",
         "twig/twig": "*",
         "knplabs/knp-markdown-bundle": "~1.3.2",
-        "egeloen/ckeditor-bundle": "~2.2",
+        "egeloen/ckeditor-bundle": "~3.0",
```

Comment:
We only need this behaviour if we really want Backward Compatibility.
IMO: If there is any ~4.0 version we should drop the support for ~2.2.